docs/modules/ROOT/pages/quarkus-opentelemetry-exporter-azure.adoc

```diff
@@ -21,16 +21,28 @@ After, add the https://mvnrepository.com/artifact/io.quarkiverse.opentelemetry.e
 
 You also need an Application Insights resource receiving the telemetry data. Go to the Azure portal, search for your Application Insights resource or create a new one. On the overview page of your resource, you will find a connection string https://learn.microsoft.com/azure/azure-monitor/app/sdk-connection-string[in the top right corner].
 
-You can set the connection string:
+You then can set the connection string:
```

Comment:
I suggest to add further clarification: 

You then can set the connection string using 2 different sets of properties, one using the Quarkus naming semantics and the other one using the default naming from the Azure documentation: